src/main/java/com/cinchapi/ccl/control/order/generated/PropagatedSyntaxException.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2013-2018 Cinchapi Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cinchapi.ccl.control.order.generated;
+
+import com.cinchapi.ccl.control.order.Parser;
+import com.cinchapi.ccl.SyntaxException;
+import com.cinchapi.common.base.AnyStrings;
+
+/**
+ * An unchecked exception that wraps a propagated exception and adds context
+ * about the parser state.
+ * 
+ * @author Jeff Nelson
+ */
+@SuppressWarnings("serial")
+public class PropagatedSyntaxException extends SyntaxException {
```

Comment:
Can we use the same version of this that already exists?

Reply:
We can't use the same version unmodified, but I could make a version that takes a string instead of a Parser that way either parser could just pass in the input string. Or make a BaseParser class that has a .input() method that the PropagatedSyntaxException can call regardless of it it's Parser or OrderParser